lib/errors.ts

```diff
@@ -122,6 +122,22 @@ const messages: {
 
 	EACCES: (e) => messages.EPERM(e),
 
+	BalenaSettingsPermissionError: () => stripIndent`
```

Comment:
I'm not sure if we need to write tests for it.

Reply:
I think testing manually would be OK in this case. The CLI's `package.json` mentions both `balena-errors` and `balena-settings-storage`, so may need updating after balena-io-modules/balena-settings-storage/pull/29 is merged.

Reply:
I did another manual test after updating my branch to the latest master and I am getting 
`Login required: use the “balena login” command to log in.` instead of the custom error message. I'm not sure why.

I will investigate it further later.

Reply:
`balena-auth` still with the old version of `balena-settings-storage`. That's why.

Reply:
Check this PR's diff for the `npm-shrinkwrap.json` file: while the CLI is using `balena-settings-storage` v7, `balena-auth` still requires `balena-settings-storage` v6. Both versions v6 and v7 will be found in the CLI's `node_modules` folder, and `balena-auth` will use v6, not v7. Perhaps a pull request is also needed on the `balena-auth` repo, to update its `package.json`: https://github.com/balena-io-modules/balena-auth/blob/v4.0.2/package.json#L66